src/.nuget/Microsoft.NETCore.Jit/win/Microsoft.NETCore.Jit.pkgproj

```diff
@@ -9,9 +9,13 @@
   </PropertyGroup>
   <ItemGroup>
     <ArchitectureSpecificNativeFile Include="$(BinDir)clrjit.dll" />
+    <CrossArchitectureSpecificNativeFile Include="$(BinDir)$(CrossTargetComponentFolder)\clrjit.dll" />
     <File Include="@(ArchitectureSpecificNativeFile)">
       <TargetPath>runtimes/$(PackageTargetRuntime)/native</TargetPath>
     </File>
+    <File Condition="'$(HasCrossTargetComponents)' == 'true'" Include="@(CrossArchitectureSpecificNativeFile)">
+      <TargetPath>runtimes/$(CrossTargetComponentFolder)_$(PackagePlatform)/native</TargetPath>
```

Comment:
I think it's a policy about how we layout binaries in the package. All consumers should set right dependencies or copying after downloading/expanding it. 

For instance, I think crossgen is an executable (tool) so it's under `tools`. The cross-target of crossgen is under its subtree. In either case, it requires to set or copy an appropriate coreclr.dll/clrjit.dll which is laid under `runtimes`.
For consistency, I place this clrjit.dll under `runtimes`. All consumers (either crossgen or corert) should place this before consuming it.